Networking/Networking/Model/OrderFeeLine.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+
+/// Represents a FeeLine Entity within an Order.
+///
+public struct OrderFeeLine: Decodable {
+    public let feeID: Int64
+    public let name: String
+    public let taxClass: String
+    public let taxStatus: OrderFeeTaxStatus
+    public let total: String
+    public let totalTax: String
+    public let taxes: [OrderItemTax]
+    public let attributes: [OrderItemAttribute]
+
+    /// OrderFeeLine struct initializer.
+    ///
+    public init(feeID: Int64,
+                name: String,
+                taxClass: String,
+                taxStatus: OrderFeeTaxStatus,
+                total: String,
+                totalTax: String,
+                taxes: [OrderItemTax],
+                attributes: [OrderItemAttribute]) {
+        self.feeID = feeID
+        self.name = name
+        self.taxClass = taxClass
+        self.taxStatus = taxStatus
+        self.total = total
+        self.totalTax = totalTax
+        self.taxes = taxes
+        self.attributes = attributes
+    }
+}
+
+
+/// Defines all of the OrderFeeLine's CodingKeys.
+///
+private extension OrderFeeLine {
+
+    enum CodingKeys: String, CodingKey {
+        case feeID      = "id"
+        case name       = "name"
+        case taxClass   = "tax_class"
+        case taxStatus  = "tax_status"
+        case total      = "total"
+        case totalTax   = "total_tax"
+        case taxes      = "taxes"
+        case attributes = "meta_data"
+    }
+}
+
+
+// MARK: - Comparable Conformance
+//
+extension OrderFeeLine: Comparable {
+    public static func == (lhs: OrderFeeLine, rhs: OrderFeeLine) -> Bool {
+        return lhs.feeID == rhs.feeID &&
+            lhs.name == rhs.name &&
+            lhs.total == rhs.total &&
+            lhs.totalTax == rhs.totalTax
+    }
```

Comment:
nit: do we intentionally choose this custom `Equatable` implementation, or does it make sense to conform to `Equatable` with the default implementation which compares all attributes? In other models, we generally use the default `Equatable` implementation so I was just curious when we choose to compare just a subset of attributes